### DISCUSSION TITLE
Good first issue

### DISCUSSION THREAD

**Discussion by devtooligan** [crytic/slither#2006] [General]
If anyone wants to take this.  Would be helpful and we can support you in working through it.

https://github.com/crytic/slither/issues/1034

**Comment by LouisTsai-Csie** [comment]
Hello, I can assist in the issue. I am new to open source and slither, but I have some development experience and the fundamentals of Solidity. Please tell me where I can start! Thank you.

**Reply by devtooligan** [reply]
Great thank you.  It looks like for this one the solution has already been suggested [here](https://github.com/crytic/slither/issues/1034#issuecomment-1483697733).

So you can clone the repo. Make the change. You can create a example contract to run your fixed detector on and see if it worked.  If it did then you can create a pull request.
